extension/src/store/actions.js

```diff
@@ -77,6 +77,18 @@ export default ({ apollo }) => {
     )
   }
 
+  const deleteAccount = ({ commit }, {address, password}) => {
+    chrome.runtime.sendMessage(
+      {
+        type: 'DELETE_WALLET',
+        payload: { address, password }
+      },
+      function (response) {
+        commit('setAccounts', response || [])
+      }
```

Comment:
this one should also call `forgetAccount`, shouldn't it?

Reply:
you would probably receive the new accounts list from background.js and just set it